scripts/codex/guard.ps1

```
@@ -0,0 +1,312 @@
+
+    if ($token.StartsWith(":")) {
+      throw "Branch deletion refspecs are forbidden in Codex guard mode: $CommandText"
+    }
+
+    if (Test-ProtectedPushSpec -Token $token) {
+      throw "Push command targets a protected branch: $CommandText"
+    }
+  }
+}
+
+function Get-RepositoryCoordinates {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-RepositoryCoordinates' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-RepositoryCoordinates'

(PSUseSingularNouns)